contentcuration/contentcuration/frontend/channelList/views/KCard/BaseCard.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+
+  <li
+    :class="['remove-list-style card',$computedClass(coreOutlineFocus)]"
+    tabindex="0"
+    data-focus="true"
+    @focus="cardFocus"
+    @hover="cardHover"
+  >
+    <component
+      :is="headerLevel"
+    >
+      <KTextTruncator
+        v-if="title !== null"
+        :text="title"
+        :maxLines="titleLines"
+        class="spacing"
+      />
+    </component>
+    <slot name="default"></slot>
+  </li>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'BaseCard',
+    props: {
+      title: {
+        type: String,
+        required: false,
+        default: null,
+      },
+      headingLevel: {
+        type: Number,
+        required: true,
+      },
+      titleLines: {
+        type: Number,
+        required: true,
```

Comment:
[Required] This is not particularly important to provide and is therefore not required. When it's not provided it will just use the default value you defined below. So you can set to `false`.